src/routes/components/components.json

```
@@ -2676,5 +2676,14 @@
 		"category": "User Interaction",
 		"tags": ["animations", "components and libraries", "interactions"],
 		"stars": 0
-	}
+	},
+  	{
```

There are unnecessary spaces
```suggestion
	{
```